src/Google/Ads/GoogleAds/Util/V10/ResourceNames.php

```diff
@@ -105,6 +105,25 @@
  */
 final class ResourceNames
 {
+
+    /**
+     * Generates a resource name of audience.
+     *
+     * @param string $customerId
+     * @param string $audienceId
+     * @return string the audience resource name
+     */
+    public static function forAudience(
```

Comment:
@fiboknacky Did you update our code generator so that this will be generated in the next major version?

Reply:
I'll do this soon.